src/command/explain.rs

```diff
@@ -0,0 +1,35 @@
+use crate::command::RoverStdout;
+use crate::Result;
+use crate::error::metadata::code::Code;
+use structopt::StructOpt;
+use serde::Serialize;
+use std::convert::TryFrom;
+use ansi_term::Colour::{Red};
+
+#[derive(Debug, Serialize, StructOpt)]
+pub struct Explain {
+    /// The full error code. For example, E020
+    #[structopt(name = "CODE")]
+    code: String,
```

Comment:
```suggestion
    code: Code,
```

^ just to keep in line with `structopt` doing our parsing for us